src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -32,6 +35,8 @@ public Climber() {
     m_rightCaneMotor.setInverted(true);
     leftSwitch = new DigitalInput(RobotMap.CL_LEFT_LIMIT_SWITCH);
     rightSwitch = new DigitalInput(RobotMap.CL_RIGHT_LIMIT_SWITCH);
+    m_clawpiston1 = new DoubleSolenoid(RobotMap.B_PCM_CAN, PneumaticsModuleType.CTREPCM, RobotMap.CL_PISTON_PORT2, RobotMap.CL_PISTON_PORT1);
+    m_clawpiston2 = new DoubleSolenoid(RobotMap.B_PCM_CAN, PneumaticsModuleType.CTREPCM, RobotMap.CL_PISTON_PORT2, RobotMap.CL_PISTON_PORT1);
```

Comment:
The ports CL_PISTON_PORT2 and CL_PISTON_PORT1 need to be different for the different pistons.
Let's call the pistons m_leftClawPiston and m_rightClawPiston to keep track of what side of the robot each piston is on.